### PR TITLE
Disable-Enable-PCI: add generic test script and test case for 10 nvme + 7 SR-IOV NICs

### PR DESCRIPTION
**Pull request by stefanlupsa**
* Add back old SRIOV-PCI-RESCIND test case with new name
* Rename instances of rescind-pci to Disable-enable-pci (rescind refers to when the request is triggered by the host) 
* Add test script for disable-enable PCI testing of multiple configurations of NVME + SR-IOV + GPU 
* Add test case for disable-enable PCI with 10 nvme disks and 7 SR-IOV NICs

### CODE REVIEW

File: Testscripts/Linux/disableEnablePci.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    if [ "$disk_count" -ne "$expectedNvme" ]; then
+        LogErr "NVME disks count does not match expected inside the VM, expected $expectedNvme found $disk_count"
+        SetTestStateFailed
+        exit 0
+    fi
+fi
+
+if [ ! -z "$expectedSriov" ] && [ "$expectedSriov" -gt 0 ]; then
+    #count SR-IOV NICs
+    nics_count=$(find /sys/devices -name net -a -ipath '*vmbus*' | grep pci | wc -l)
```
Comment (chvalean) [inline Testscripts/Linux/disableEnablePci.sh]: https://github.com/koalaman/shellcheck/wiki/SC2126

Comment (stefanlupsa) [reply Testscripts/Linux/disableEnablePci.sh]: moved to grep -c

**Comment by stefanlupsa** [comment]
```
05/06/2019 17:13:31 : [INFO ] PCI-DEVICE-DISABLE-ENABLE-SRIOV-NVME ended running with status: PASS.
05/06/2019 17:13:31 : [INFO ] Test YZ16 finished
05/06/2019 17:13:31 : [INFO ] 
[LISAv2 Test Results Summary]
Test Run On           : 05/06/2019 16:52:22
ARM Image Under Test  : Canonical : UbuntuServer : 19.04 : latest
Total Test Cases      : 1 (1 Passed, 0 Failed, 0 Aborted, 0 Skipped)
Total Time (dd:hh:mm) : 0:0:21

   ID TestArea             TestCaseName                                                                TestResult TestDuration(in minutes) 
-------------------------------------------------------------------------------------------------------------------------------------------
    1 CORE                 PCI-DEVICE-DISABLE-ENABLE-SRIOV-NVME                                              PASS                10.31 
```

**Review by chvalean — APPROVED** [review]

**Comment by juhlee-microsoft** [comment]
PCI device count should be 8

**Pull request merged**